components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -1058,4 +1070,31 @@ private AuthenticatorConfig getAuthenticatorConfig() {
         }
         return authConfig;
     }
+
+    /**
+     * Check whether the user should be redirected to the retry.jsp page when the user's account is locked.
+     * This decision is taken based on three configuration options, redirectToMultiOptionPageOnFailure,
+     * showAuthFailureReasonOnLoginPage and redirectToRetryPageOnAccountLock.
+     *
+     * @param context  Authentication context.
+     * @return boolean Whether the user should be directed to retry.jsp page or not.
+     */
+    private boolean isRedirectionToRetryPage(AuthenticationContext context) {
+
+        boolean sendToMultiOptionPage = context.isSendToMultiOptionPage();
+        if (!sendToMultiOptionPage) {
```

Comment:
code readability can be improved if we write as,
```suggestion
        if (sendToMultiOptionPage) {
              return false;
        }
```

Then code block 1086-1096 can be left aligned

Reply:
resolved in https://github.com/wso2/carbon-identity-framework/pull/4171/commits/2f2c5b3ae4c78462563a984fa6a9f7f95edbd4d8